### PR TITLE
Fix compile http-parser-2.1 fallthrouth error above gcc7

### PR DESCRIPTION
**Pull request by lightningkay**
by the way, I format some spaces in the shell, sorry for my obession
https://gcc.gnu.org/onlinedocs/gcc-7.3.0/gcc/Warning-Options.html#index-Wimplicit-fallthrough

### CODE REVIEW

File: trunk/auto/depends.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ function Ubuntu_prepare()
             fi
         fi
     fi
-    
+
```
Comment (winlinvip) [inline trunk/auto/depends.sh]: Please revert these empty lines

**Comment by winlinvip** [comment]
Won't fix for SRS2.

**Comment by winlinvip** [comment]
It's a common error for #1065 #1159 #1220 #1154 #1129 #1159

File: trunk/auto/depends.sh (hunk 1 above)
Comment (lightningkay) [reply trunk/auto/depends.sh]: I think this spaces is unnessary.

**Pull request closed without merging**